### ISSUE TITLE
echo bridge not working on wired lan devices 

### ISSUE THREAD

**Issue by megamalain** [armzilla/amazon-echo-ha-bridge#15]
Amazon Echo will only detect devices if the echo bridge host machine is connected to the LAN wirelessly with the same SSID (e.g. no device detection if a network has 2.5GHz and 5GHz wireless networking with different SSIDs, and Amazon Echo and host are on them respectively). Also, echo bridge doesn't seem to support host machines connected to LAN wired (i.e. through ethernet cable).

**Comment by craigwitter** [comment]
I have mine on the wired network and it detects fine.

**Comment by megamalain** [comment]
@craigwitter I tried this on Pi B+ and Pi 2 with Raspian and Arch for v0.1.1,v0.1.2,v0.1.3 on wired network with no success until I used wireless networking

**Comment by okachobi** [comment]
I had a similar problem, and it turned out to be that my wireless AP was not bridging broadcast/multicast traffic between the LAN connection and Wireless interface.  Once I enabled multicast bridging, it worked flawlessly.

**Comment by markfalk** [comment]
Exactly. It’s all done via multicast. The bridge must get traffic destined for the multicast address 239.255.255.250. The echo solicits information by sending to that multicast address, if the bridge doesn’t see it, it can’t respond.

**Comment by shanekuz** [comment]
Hi have me echo and amazon-echo-ha-bridge pc on a different subnet and it still works fine, every now and then when I issue a command the echo says it could not contact the device but the device still turns on or off.  I will put them on the same subnet and see if that goes away.

**Comment by okachobi** [comment]
I believe only the initial discovery occurs over multicast.  Once the Echo discovers the bridge and queries the services, it appears to save the information and do direct TCP requests to the bridge.  I discovered my problem by moving the echo to a different wireless access point that I knew was bridging.  It was then discovered and continued to work when it connected with the AP that did not bridge.  But running the discovery again would fail.

**Comment by maddox** [comment]
I have all Airport devices for WLAN at my house:

* Airport Extreme 802.11 AC
* Airport Express 802.11 N
* Airport Express 802.11 N

All devices are set to Bridged mode, as my FIOS router handles DHCP on my network.

I can't, for the life of me, get my Echo or my Harmony hub to see the bridge. Has anyone figured this out? Should the Hue iOS app be able to see the bridge too?

I have `Enable IGMP Snooping` enabled on my Airport Extreme base station, but I've still had no luck.

**Comment by maddox** [comment]
OK, ignore me!

I had misspelled `--upnp.config.address` as `--upnp.config.adress `

It's working like it should now!!

**Comment by gbarnabic** [comment]
I had everything working fine on my ubuntu linux computer for a few months when the power supply died. Once I got it back up running again the echo bridge did not work. After a little bit of hair pulling I realized that the IP address on the LAN had changed. So I had to not only change the --upnp.config.address=192.168.1.## parameter when starting the hue echo bridge I also had to change all the device ON and OFF triggers, updating them with the new IP address. Just posting this here for others to see down the road.
